app/src/main/res/values/strings.xml

```diff
@@ -66,9 +66,10 @@
     <string name="draft">Draft</string>
     <string name="theme">Theme</string>
     <string name="pub_timeline">Public Timeline</string>
-    <string name="load_failure_explanation">Loading the data from Github failed. Please verify you have a working network connection and try again.</string>
+    <string name="load_failure_explanation">Loading the data from GitHub failed. Please verify you have a working network connection and try again.</string>
     <string name="load_failure_explanation_dmca">This repository was taken down for legal reasons.\nPlease see %1$s for details.</string>
-    <string name="load_failure_explanation_with_reason">Loading the data from Github failed. Github told us the following reason:\n%1$s</string>
+    <string name="load_failure_explanation_parsing">There was an error while parsing GitHub\'s response. See the logcat for details.</string>
```

Comment:
I'm not sure whether showing this message really makes sense with that wording:
- Even though the target audience of OctoDroid is technical, can we reasonably expect all of them to know the term 'logcat'?
- 'logcat' is the log viewer tool, not the log itself ;-)
- Shouldn't we rather tell them to open an issue and attach the log?

For the latter, I think a precondition would be implementing some log collection (spawn logcat process and collect its output into string array).

Reply:
> Shouldn't we rather tell them to open an issue and attach the log?

That's a good idea. I wrote that error message as a less daunting alternative to printing the exception message straight to the user 😅 (which is actually what we care about in those cases).
I think that we don't need to access the log via logcat, given that we have access to the full stack trace. At this point I see two alternatives for the error screen:
- the lazy one: tell the user to report the error and add a button to copy the exception stack trace to the clipboard
- the awesome-UX one: add a button that goes straight to the "New issue" activity for the OctoDroid repo and prepopulates the issue body with a short description and the exception stack trace (maybe slightly overkill?)

What do you think?

Reply:
Given the second option is close to no overhead (since all that needs to be added is a new intent extra in `IssueEditActivity` for title + body), I'm leaning towards the latter. Can repurpose the 'retry' button in that case :-)

Reply:
Done! A couple of notes:
- I haven't pre-filled the issue title for error reports, since this forces the user to try to write a meaningful title for the specific situation
- in the pre-filled issue body I haven't added any device specs, since `JsonDataException`s aren't device or OS-dependent. We can add them later if the "Report issue" features is used in other scenarios
- when the issue report is submitted, we will just display a "Thank you" snackbar (not the created issue). Is this good enough?

Reply:
> when the issue report is submitted, we will just display a "Thank you" snackbar (not the created issue). Is this good enough?

I think so. One could use the snackbar action to go to it ... this would mean we'd need to return e.g. the issue number in the activity result.

Reply:
Done, I've added a 'View' action to the snackbar